meta-refkit/lib/oeqa/selftest/iotsstatetests.py

```diff
@@ -33,7 +33,8 @@ def test_sstate_samesigs(self):
         libcappend = get_bb_var('TCLIBCAPPEND')
         # Select subset of the machines to speed up testing.
         # Edison/intel-core2-32 are particularly sensitive.
-        machines = "edison intel-quark intel-core2-32 intel-corei7-64 beaglebone".split()
+        machines = "intel-corei7-64".split()
```

Comment:
I asked in bugz. ticket does it make sense to enable this test in refkit, and you replied "in principle it might still be useful", so I guessed there may be something usable even with one machine. But other than that, yes, it's used as activation trial now, plus it will be ready to strike after someone adds more MACHINE targets.